programs/mango-v4/src/lib.rs

```diff
@@ -1125,6 +1125,93 @@ pub mod mango_v4 {
         Ok(())
     }
 
+    ///
+    /// OpenbookV2
+    ///
+
+    pub fn openbook_v2_register_market(
+        ctx: Context<OpenbookV2RegisterMarket>,
+        market_index: OpenbookV2MarketIndex,
+        name: String,
+    ) -> Result<()> {
+        Ok(())
+    }
+
+    pub fn openbook_v2_edit_market(
+        ctx: Context<OpenbookV2EditMarket>,
+        reduce_only_opt: Option<bool>,
+        force_close_opt: Option<bool>,
+    ) -> Result<()> {
+        Ok(())
+    }
+
+    pub fn openbook_v2_deregister_market(ctx: Context<OpenbookV2DeregisterMarket>) -> Result<()> {
+        Ok(())
+    }
+
+    pub fn openbook_v2_create_open_orders(ctx: Context<OpenbookV2CreateOpenOrders>) -> Result<()> {
```

Comment:
create open orders must take at least `account_num: u32, open_orders_count: u8` to match the accounts struct

Reply:
(and I don't yet understand what open_orders_count is for)

Reply:
Yes not needed `open_orders_count`.